interp/interp.go

```diff
@@ -330,26 +329,38 @@ func (interp *Interpreter) main() *node {
 	return nil
 }
 
-// EvalInc is Eval in incremental mode, with the default input name.
-func (interp *Interpreter) EvalInc(src string) (res reflect.Value, err error) {
-	return interp.Eval(src, "", true)
+// Eval evaluates Go code represented as a string. Eval returns the last result
+// computed by the interpreter, and a non nil error in case of failure.
+func (interp *Interpreter) Eval(src string) (res reflect.Value, err error) {
+	return interp.eval(src, "", true)
 }
 
-// Eval evaluates Go code represented as a string. It returns a map on
-// current interpreted package exported symbols.
-// Name is the file path to the Go source file containing src. It defaults to the
-// special value "_.go".
-// Incremental is whether the given source should be considered an integral
-// valid Go source file, or only a fragment.
-func (interp *Interpreter) Eval(src, name string, incremental bool) (res reflect.Value, err error) {
+// EvalName evaluates Go code represented as a string. Name is the import path of code
+// and can be left to "" for main. EvalName returns the last result computed by the
+// interpreter, and a non nil error in case of failure.
+func (interp *Interpreter) EvalName(src, name string) (res reflect.Value, err error) {
+	return interp.eval(src, name, true)
```

Comment:
Why is EvalName always in incremental mode?
And if it is, I think it's weird that some of the Eval* methods are in incremental mode, whereas some others are not, and that we haven't even documented it for the user.